### ISSUE TITLE
Icons are not rendering

### ISSUE THREAD

**Issue by mrcasals** [react-native-material-design/react-native-material-design#3]
This is a screenshot of an avatar:

![](https://i.imgur.com/4e0xsdb.png)

This is what I'm using to render it:

```html
      <Avatar
        icon={'face'}
        backgroundColor={this.getBackgroundColor()}
      />
```

I've made sure I copied the correct font file and restarted the package, but it still doen't load. Any idea?

Thanks!

**Comment by Ehesp** [comment]
Hmm strange, try a few others... Potentially the icon package might be out
of date and not have some newer ones.

**Comment by mrcasals** [comment]
If I enter a random string, the warning shows a list of all available
icons. "face" is there :/

On Fri, Dec 18, 2015 at 5:42 PM Elliot Hesp <notifications@github.com>
wrote:

> Hmm strange, try a few others... Potentially the icon package might be out
> of date and not have some newer ones.
> On 18 Dec 2015 17:16, "Marc Riera" <notifications@github.com> wrote:
>
> > This is a screenshot of an avatar:
> >
> >
> > <
> https://camo.githubusercontent.com/43611032cb80f83237dc7ae33daa11e1dcad80df/68747470733a2f2f692e696d6775722e636f6d2f346530787364622e706e67
> >
> >
> > This is what I'm using to render it:
> >
> > <Avatar
> > icon={'face'}
> > backgroundColor={this.getBackgroundColor()}
> > />
> >
> > I've made sure I copied the correct font file and restarted the package,
> > but it still doen't load. Any idea?
> >
> > Thanks!
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/react-native-material-design/react-native-material-design/issues/3
> >
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/react-native-material-design/react-native-material-design/issues/3#issuecomment-165830620>
> .
>
-- 
Marc Riera

**Comment by Ehesp** [comment]
Hmm okay, which version of react native are you running? If you check in the build.gradle file in the android dir it'll tell you the exact version.

**Comment by mrcasals** [comment]
Oh, it looks like I missed fixing this file when upgrading to RN 0.16,
sorry. Will check if this is the source of the bug.

On Fri, Dec 18, 2015 at 5:49 PM Elliot Hesp <notifications@github.com>
wrote:

> Hmm okay, which version of react native are you running? If you check in
> the build.gradle file in the android dir it'll tell you the exact version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/react-native-material-design/react-native-material-design/issues/3#issuecomment-165834463>
> .
>
-- 
Marc Riera

**Comment by Ehesp** [comment]
Yeah 0.16 introduced auto font loading, also make sure the directory is called 'fonts' (note all lower case) too.

**Comment by mrcasals** [comment]
Hi, I rechecked it and I'm really using the correct RN version in the build.gradle file:

```
compile "com.facebook.react:react-native:0.16.+"
```

**Comment by Ehesp** [comment]
Strange then. If you head over to
https://github.com/oblador/react-native-vector-icons try running through
the installation guide on there, it should be getting loaded in for you.

**Comment by dgilperez** [comment]
I'm having the same problem. Same version of RN and latest version of react-native-vector-icons. I followed up the installation guide a couple of times, no success. @mrcasals did you solve the issue?

**Comment by Ehesp** [comment]
Okay I'll look at this tomorrow - I've got it working myself so hopefully something simple.

**Comment by Ehesp** [comment]
Icons work here on both 0.16 & 0.17... However I did upgrade the vector icons package (https://github.com/oblador/react-native-vector-icons/releases/tag/v1.0.3). Maybe that was causing the problem. The latest release (0.2.1) has this in it, so if you could give that a go, ta.

**Comment by bozzmob** [comment]
Considering its fixed post 0.2.1, will be closing this issue.

**Comment by xotahal** [comment]
Have anyone problem with this?

My versions:
react-native-material-design: 0.3.2
react-native-vector-icons: 1.2.0

I copied fonts and tried the instal of react-native-vector-icons - no success
